ocaml/utils/load_path.ml

```diff
@@ -18,35 +18,39 @@ module Dir : sig
   type t
 
   val path : t -> string
-  val files : t -> string list
+  val files : t -> (string * string) list
```

Comment:
The meaning of `(string * string)` is not obvious here. Maybe define a small type for it?